addons/mrp/models/mrp_production.py

```diff
@@ -400,17 +401,20 @@ def _compute_delay_alert_date(self):
     def _compute_json_popover(self):
         production_no_alert = self.filtered(lambda m: m.state in ('done', 'cancel') or not m.delay_alert_date)
         production_no_alert.json_popover = False
+        production_no_alert.planning_issues = 0
         for production in (self - production_no_alert):
-            production.json_popover = json.dumps({
-                'popoverTemplate': 'stock.PopoverStockRescheduling',
-                'delay_alert_date': format_datetime(self.env, production.delay_alert_date, dt_format=False),
-                'late_elements': [{
+            late_elms = [{
                         'id': late_document.id,
                         'name': late_document.display_name,
                         'model': late_document._name,
                     } for late_document in production.move_raw_ids.filtered(lambda m: m.delay_alert_date).move_orig_ids._delay_alert_get_documents()
                 ]
+            production.json_popover = json.dumps({
```

Comment:
You still display the json_popover field somewhere? Since you are now using planning_issues in the view

Reply:
planning_issues is just used in the listview, json_popover is still used in the formview. This is to hide the JSON value being displayed when hovering the field in listview